entity/src/main/java/com/fiap/burger/entity/order/Order.java

```diff
@@ -143,4 +143,8 @@ public Order(
         this.modifiedAt = modifiedAt;
         this.deletedAt = deletedAt;
     }
+
+    public static boolean canBePaid(OrderStatus status) {
+        return OrderStatus.AGUARDANDO_PAGAMENTO.equals(status);
+    }
```

Comment:
Não deveria ser o status do pedido a ser considerado? Ao invés de usar um status vindo por parâmetro?
```suggestion
    public static boolean canBePaid() {
        return OrderStatus.AGUARDANDO_PAGAMENTO.equals(this.status);
    }
```